### PR TITLE
Fix Validator For Spadina

### PR DESCRIPTION
**Pull request by nisdas**
**What type of PR is this?**

Bug Fix

**What does this PR do? Why is it needed?**

- [x] Spadina was not declared anywhere for the validator v2 subcommands. This PR takes care of that.

**Which issues(s) does this PR fix?**

**Other notes for review**

### CODE REVIEW

File: validator/accounts/v2/cmd_accounts.go [hunk 1]
```diff
@@ -26,9 +26,10 @@ this command outputs a deposit data string which is required to become a validat
 				flags.WalletPasswordFileFlag,
 				flags.AccountPasswordFileFlag,
 				flags.NumAccountsFlag,
+				flags.DeprecatedPasswordsDirFlag,
 				featureconfig.AltonaTestnet,
 				featureconfig.OnyxTestnet,
-				flags.DeprecatedPasswordsDirFlag,
```
Comment (farazdagi) [inline validator/accounts/v2/cmd_accounts.go:31]: Is it intended (flag that is not related to PR is being removed)?

**Review by farazdagi — COMMENTED** [review]
Just a minor comment.

File: validator/accounts/v2/cmd_accounts.go (hunk 1 above)
Comment (nisdas) [reply validator/accounts/v2/cmd_accounts.go:31]: Not being removed, its just shifted up if you look above.

**Review by farazdagi — APPROVED** [review]

File: validator/accounts/v2/cmd_accounts.go (hunk 1 above)
Comment (farazdagi) [reply validator/accounts/v2/cmd_accounts.go:31]: Ok, my bad. Approving.
